aerogear/aerogear-android-cookbook
# Sync demo

How to test

1. Clone and build locally [Android Sync Library](https://github.com/aerogear/aerogear-android-sync/tree/master/aerogear-android-sync)
1. Run the [Sync Server (JSON Patch)](https://github.com/aerogear/aerogear-sync-server)
1. Configure the Application
   1. Set serverHost in AndroidManifest.xml to an appropriate host. The default is for the emulator host's IP (so if you run this in the Android emulator you will get the desired behavior)
1. Run the app (./gradlew installDebug)
1. Edit the doc then Save
1. Close the app (or even uninstall)
1. run the app again
1. you should see your changes

Bonus testing!

Multiple devices. You should see changes on save propagate.
